src/jquery.nepaliDatePicker.js

```diff
@@ -8,8 +8,8 @@
 var calendarFunctions = {};
 (function ($) {
     var calendarData = {
-        bsMonths: ["बैशाख", "जेठ", "असार", "सावन", "भदौ", "असोज", "कार्तिक", "मंसिर", "पौष", "माघ", "फागुन", "चैत"],
```

Comment:
On second look `बैशाख` seems to be correct as well. At least `बैशाख` is how we say it. Nepali is hard. 😓

Reply:
Hahaha. I'm feeling like a language-Nazi right now.

Reply:
`बैशाख` seems to be more prevalent. Let's go with that.